### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by sergiy-petrov**
add php 7.3 and 7.4 to travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,14 @@ matrix:
...
   - php: 7.2
     env: SWAGGER_VERSION=3.0 CACHE_NAME=OPEN_API REPORT_TESTS_COVERAGE=1
+  - php: 7.3
+    env: CACHE_NAME=SWAGGER
+  - php: 7.3
+    env: SWAGGER_VERSION=3.0 CACHE_NAME=OPEN_API REPORT_TESTS_COVERAGE=1
+  - php: 7.4
+    env: CACHE_NAME=SWAGGER
+  - php: 7.4
+    env: SWAGGER_VERSION=3.0 CACHE_NAME=OPEN_API REPORT_TESTS_COVERAGE=1
```
Comment (DarkaOnLine) [inline .travis.yml]: Only one php version should run with `REPORT_TESTS_COVERAGE=1`

Comment (sergiy-petrov) [reply .travis.yml]: @DarkaOnLine so which one? 7.4?

Comment (DarkaOnLine) [reply .travis.yml]: Have you tested this package on 7.4? If so and it works - let's do 7.4 ;)

Comment (sergiy-petrov) [reply .travis.yml]: @DarkaOnLine well, I haven't but I try to test in a few days

**Pull request merged**